src/js/signin/signin-entry.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { Provider } from 'react-redux';
+
+import initReact from '../common/init-react';
+import createCommonStore from '../common/store';
+import reducer from './reducers/login';
+
+import Main from './containers/Main';
+
+require('../common');  // Bring in the common javascript.
+require('../../sass/login.scss');
+
+const commonStore = createCommonStore(reducer);
+
+export default function createLoginWidget(store, rootElementId = 'login-root') {
```

Comment:
Isn't `login-root` in the top nav? How does this avoid plopping the whole page into the header?

Reply:
There's a parameter where I pass in `react-root`

https://github.com/department-of-veterans-affairs/vets-website/blob/61a45258cbe0dafd3f9b2ee58ec19792eacdab65/src/js/signin/signin-entry.jsx#L29

maybe it's cleaner to hard code this in the function

Reply:
Oh, got it. Missed that.